app/screens/auth/sign_up_screen.tsx

```diff
@@ -19,7 +19,7 @@ export default function SignUp() {
   const [email, setEmail] = React.useState("");
   const [password, setPassword] = React.useState("");
   const [confirmPassword, setConfirmPassword] = React.useState("");
-  const router = useRouter();
+  const router = App();
```

Comment:
There are specific use cases for the navigation in the documentation where they handle authentification